t/react_template/cypress/support/e2e.ts

```diff
@@ -0,0 +1,14 @@
+// ***********************************************************
```

Comment:
Let's also remove this file

Reply:
e2e.ts is default requirement in cypress. Suggest to keep it